src/TwigComponent/src/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Symfony\UX\TwigComponent\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder('twig_component');
+        $rootNode = $treeBuilder->getRootNode();
+        $rootNode
+            ->children()
+                ->scalarNode('template_extension')->defaultValue('.html.twig')->end()
```

Comment:
I think we should drop this config option (for now). If we use some form of the chain I suggest, this should not be needed anyway.

Probably not realistic but could be a problem if a 3rd party bundle expects `.html.twig` but you've customized it in your app.

Reply:
I still think we should drop the config to start.